test/csuite/schema_abort/main.c

```diff
@@ -509,6 +510,18 @@ get_all_committed_ts(void)
     return (ret);
 }
 
+/*
+ * is_set_timestamp_required --
+ *     Return if set_timestamp is required.
+ */
+static bool
+is_set_timestamp_required(uint64_t last_ts, uint64_t oldest_ts)
+{
+    int64_t ts_diff = (int64_t)(oldest_ts - last_ts);
```

Comment:
We should compare the `uint64_t` values and not cast them, we risk an overflow bug here.

Reply:
@etienneptl The intention here is to check that the `current_oldest_ts` is greater than `last_oldest_ts`.  I have an alternative approach to determine if `current_oldest_ts` exceeds `last_oldest_ts` that does not involve calculating their difference.

`
    return ((oldest_ts == stop_timestamp && oldest_ts != last_ts) ||
      (oldest_ts != UINT64_MAX && oldest_ts > last_ts && oldest_ts - last_ts > STABLE_PERIOD));

`

Reply:
Yes, it's better this way, thanks!